### PR TITLE
Fixup updateSnapshotInfo failure due to forward merging

### PR DESCRIPTION
**Pull request by vishesh92**
### Description

This PR...

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]
LGTM - didn't test it

File: server/src/test/java/com/cloud/storage/snapshot/SnapshotManagerImplTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public class SnapshotManagerImplTest {
     @Mock
     SnapshotService snapshotService;
     @Mock
-    SnapshotDataFactory snapshotDataFactory;
+    SnapshotDataFactory snapshotFactory;
```
Comment (shwstppr) [inline server/src/test/java/com/cloud/storage/snapshot/SnapshotManagerImplTest.java:77]: Are changes here needed?

Comment (vishesh92) [reply server/src/test/java/com/cloud/storage/snapshot/SnapshotManagerImplTest.java:77]: Yes. Getting a null pointer exception.
https://github.com/apache/cloudstack/actions/runs/6651356366/job/18073228134

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Review by weizhouapache — APPROVED** [review]
code lgtm

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by harikrishna-patnala — APPROVED** [review]
Code LGTM

**Pull request merged**